tools/flang2/flang2exe/cgmain.cpp

```diff
@@ -982,51 +1004,51 @@ cons_loop_parallel_accesses_metadata(void)
   return ll_get_md_node(cpu_llvm_module, LL_PlainMDNode, lvcomp, 2);
 } // cons_loop_parallel_accesses_metadata
 
+/**
+   \brief Construct exactly one cached instance of !{!"llvm.loop.vectorize.enable", 0}.
+ */
 INLINE static LL_MDRef
 cons_novectorize_metadata(void)
 {
   LL_MDRef lvcomp[2];
-  LL_MDRef loopVect;
-  LL_MDRef rv;
-
-  if (cpu_llvm_module->loop_md)
-    return cpu_llvm_module->loop_md;
-  rv = ll_create_flexible_md_node(cpu_llvm_module);
-  lvcomp[0] = ll_get_md_string(cpu_llvm_module, "llvm.loop.vectorize.enable");
-  lvcomp[1] = ll_get_md_i1(0);
-  loopVect = ll_get_md_node(cpu_llvm_module, LL_PlainMDNode, lvcomp, 2);
-  ll_extend_md_node(cpu_llvm_module, rv, rv);
-  ll_extend_md_node(cpu_llvm_module, rv, loopVect);
-  cpu_llvm_module->loop_md = rv;
-  return rv;
+  if (LL_MDREF_IS_NULL(cached_vectorize_disable_metadata)) {
+    lvcomp[0] = ll_get_md_string(cpu_llvm_module, "llvm.loop.vectorize.enable");
+    lvcomp[1] = ll_get_md_i1(0);
+    cached_vectorize_disable_metadata = ll_get_md_node(cpu_llvm_module,
+      LL_PlainMDNode, lvcomp, 2);
```

Comment:
Nit: Missing two spaces. Please use the following style.
```
    cached_vectorize_disable_metadata = ll_get_md_node(cpu_llvm_module,
        LL_PlainMDNode, lvcomp, 2);
```

Reply:
It's probably easier to run this hunk through clang-format, which would give:
```c
    cached_vectorize_disable_metadata =
        ll_get_md_node(cpu_llvm_module, LL_PlainMDNode, lvcomp, 2);
```